### DISCUSSION TITLE
Welcome to fcc-forum-clone Discussions!

### DISCUSSION THREAD

**Discussion by jdwilkin4** [jdwilkin4/fcc-forum-clone#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sojinsamuel** [comment]
Hie, got into serious programming in January, and found out about this community from one of my friends @dgupta0 
I'm still understanding the motive of the community & the code base. I will make sure to contribute my time and knowledge to build this community.

Let's all connect sometime together (if possible) through zoom/gmeet.

Happy coding ;)

**Comment by dgupta0** [comment]
Hi @sojinsamuel, 

Welcome to the community.  You can join our discord community through the link - https://discord.com/invite/QvvUtaThAz

Happy Coding and Learning,
